k8s/upgrades/0.9.0-1.0.0/cstor-target-patch.tpl.json

```diff
@@ -0,0 +1,27 @@
+{
+   "metadata": {
+      "labels": {
+         "openebs.io/version": "@target_version@"
+      }
+   },
+   "spec": {
+      "template": {
```

Comment:
we need to patch `spec.template.metadata.labels` also

Reply:
Done